<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LaunchMethodEditorPaneClass</class>
 <widget class="QWidget" name="LaunchMethodEditorPaneClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlaneEditorPane</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="verticalSpacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Name:</string>
     </property>
     <property name="buddy">
      <cstring>nameInput</cstring>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Co&amp;mments:</string>
     </property>
     <property name="buddy">
      <cstring>commentsInput</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Short name:</string>
     </property>
     <property name="buddy">
      <cstring>shortNameInput</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Logbook label:</string>
     </property>
     <property name="buddy">
      <cstring>logStringInput</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Keyboard shortcut:</string>
     </property>
     <property name="buddy">
      <cstring>keyboardShortcutInput</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&amp;Type:</string>
     </property>
     <property name="buddy">
      <cstring>typeInput</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="nameInput">
     <property name="toolTip">
      <string>&lt;html&gt;The name of the launch method is displayed in the flight dialog.&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="shortNameInput">
     <property name="toolTip">
      <string>&lt;html&gt;The short name of the launch method is displayed in the flight table.&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="10" column="2">
    <widget class="QLineEdit" name="commentsInput"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&amp;Person required:</string>
     </property>
     <property name="buddy">
      <cstring>personRequiredInput</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="SkComboBox" name="personRequiredInput">
        <property name="toolTip">
         <string>&lt;html&gt;If true, a warning will be output if a person is not specified for a flight with this launch method.&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>148</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="SkComboBox" name="typeInput"/>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="logStringInput">
     <property name="toolTip">
      <string>&lt;html&gt;The logbook label of the launch method is used for pilot logbook entries.&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="keyboardShortcutInput">
     <property name="toolTip">
      <string>&lt;html&gt;The keyboard shortcut can be used for quickly selecting a launch method in the flight editor.&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Towplane:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QRadioButton" name="individualTowplaneInput">
     <property name="toolTip">
      <string>&lt;html&gt;A towplane can be entered for every flight that is created with this launch method. This can be used for towplanes that are infrequently used at this airfield.&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Enter &amp;when creating a flight</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <attribute name="buttonGroup">
      <string>towplaneButtonGroup</string>
     </attribute>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QWidget" name="specificTowplanePane" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="margin">
       <number>0</number>
      </property>
      <item>
       <widget class="QRadioButton" name="specificTowplaneInput">
        <property name="toolTip">
         <string>&lt;html&gt;The specified towplane is used for flights created with this launch method. The registration does not have to be entered for every flight. This can be used for towplanes that are frequently used at this airfield.&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string>Specify &amp;registration:</string>
        </property>
        <attribute name="buttonGroup">
         <string>towplaneButtonGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="SkComboBox" name="towplaneRegistrationInput">
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="insertPolicy">
         <enum>QComboBox::NoInsert</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QComboBox::AdjustToContents</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SkComboBox</class>
   <extends>QComboBox</extends>
   <header>src/gui/widgets/SkComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>nameInput</tabstop>
  <tabstop>shortNameInput</tabstop>
  <tabstop>logStringInput</tabstop>
  <tabstop>keyboardShortcutInput</tabstop>
  <tabstop>typeInput</tabstop>
  <tabstop>individualTowplaneInput</tabstop>
  <tabstop>specificTowplaneInput</tabstop>
  <tabstop>towplaneRegistrationInput</tabstop>
  <tabstop>personRequiredInput</tabstop>
  <tabstop>commentsInput</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="towplaneButtonGroup"/>
 </buttongroups>
</ui>
